sdkproject/Assets/Mapbox/Unity/Utilities/VectorEntity.cs

```diff
@@ -0,0 +1,15 @@
+using Mapbox.Unity.MeshGeneration.Components;
```

Comment:
Now that we have vector entity, and it contains `Mesh`, can we streamline the interface for `Modifiers`? In other words, mesh and game object modifiers receive the same params?